deploy/cluster_role.yaml

```diff
@@ -14,8 +14,14 @@ rules:
   - ""
   resources:
   - nodes
+  - namespaces
```

Comment:
@nurali-techie, we should not give permission to our sa to do create/delete operations for nodes. this way, if our sa gets compromised, it has capability to take entire cluster down. So we should add it under separate block
```
- apiGroups:
   - ""
  resources:
  - namespaces
  verbs:
  - get
  - create
  - update
  - list
  - watch
  - delete

Reply:
thanks @dipak-pawar will do.

Reply:
done https://github.com/codeready-toolchain/member-operator/pull/53/commits/acfd7ab49241f7782a2259da3f2d108cc6afc426